example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -99,31 +98,24 @@ bool offb_ctrl_ned(Device &device)
     // NOTE: Use sleep_for() after each velocity-ned command to closely monitor their behaviour.
```

Comment:
Section above is confusing to watch and the code is a bit complicated:
- The vehicle does not "go right". It turns to face east unless it is already facing east.
- It technically does "oscillate", but that could mean almost anything.
- Propose change text to "Oscillate in a North-South path while facing East (turning if necessary)"
- Propose make this do two cycles to be easier to observe when running the example (ie. ` const unsigned steps = (unsigned)(one_cycle / step_size);` to `const unsigned steps = 2 * (unsigned)(one_cycle / step_size);`
- Maybe "Done" is OK here because the action is quite extended.

Reply:
Thanks, changed it along these lines.